src/root/body/index.tsx

```diff
@@ -117,6 +117,14 @@ const Body: React.FC<params> = (props) => {
     }
   };
 
+  const getBackendUrl = (): string => {
+    const backendURL = process.env.REACT_APP_BACKEND_URL;
+    if (backendURL === null || backendURL === undefined || backendURL === "") {
+      throw new Error("REACT_APP_BACKEND_URL·is·not·set·in·.env");
```

Comment:
it`s not necessarily .env.... it's an environment variable that gets loaded from either .env or ... set in the environment. don't refer it as .env